build.sbt

```diff
@@ -131,15 +131,15 @@ lazy val commonSettings = Seq(
     %%("doobie-core", V.doobie),
     %%("doobie-postgres", V.doobie),
     %%("doobie-hikari", V.doobie),
-    "com.beachape" %% "enumeratum" % V.enumeratum,
-    "com.beachape" %% "enumeratum-circe" % V.enumeratumCirce,
-    "org.flywaydb" % "flyway-core" % V.flyway,
+    "com.beachape"                    %% "enumeratum" % V.enumeratum,
```

Comment:
Is this indentation intentional? Or just automatic scalafmt?

Reply:
TBH, I don't know... It seems to be automatic scalafmt, I'll try to run it locally again

Reply:
Automatic scalafmt confirmed